### ISSUE TITLE
Does minigraph build genome graph including structural variants from a SV caller?

### ISSUE THREAD

**Issue by clairemerot** [lh3/minigraph#22]
Hello,
I'm also keen to integrate into the genome graph the structural variability detected by Sv callers (from short reads and/or long-reads). I was wondering if minigraph handles that? 
Those are often represented in a vcf which some programs can integrate into a genome-graph. I am using vg at the moment (https://github.com/vgteam/vg) which is fast at constructing the graph but slow at mapping. Hence, we are super interested in using minigraph, as it seems to be very efficient (and I remember the efficiency of minimap!). I may also have a second reference assembly so the idea in the end would be to build a graph integrating several assemblies and local SV; 
Thank you for your help and attention,
Claire

**Comment by cistarsa** [comment]
I am also attempting this, did you figure this out?

**Comment by lh3** [comment]
No, minigraph doesn't do that. To a large extent, minigraph itself is a SV caller for long contigs.

**Comment by clairemerot** [comment]
Thanks for your reply,
And thanks a lot the addition in the readme about SV-calling with minigraph, I'm trying that, as I have two assemblies to compare!

I was wondering whether it would be possible (and would make sense), to build a graph with two assemblies in minigraph, to get a good basis of pangenome graph, and then export the gfa to enrich it with more SVs (likely shorter ones inferred from larger sample sizes) in other software like vg? anyone has tried that?

Cheers
Claire
